### PR TITLE
Also export PROJECT_INCLUDE_DIRS

### PR DESCRIPTION
**Pull request by jcarpent**
@wxmerkt This is solving the bug mentionned in https://github.com/stack-of-tasks/eigenpy/issues/174

### CODE REVIEW

**Comment by nim65s** [comment]
I think the right way of doing this is to put those directories in the corresponding export.

```
find_package(eigenpy REQUIRED)
message(WARNING ${eigenpy_INCLUDE_DIRS})
```
This is not an issue by itself. The issue is that your project that depends on eigenpy doesn't get the right path to the includes. This can be fixed like in eg. https://github.com/stack-of-tasks/pinocchio/pull/1098 :

With this kind of `TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PUBLIC $<INSTALL_INTERFACE:include>)`, one can use:

```
find_package(eigenpy REQUIRED)
add_library(my_lib SHARED ${MY_SOURCES})
target_link_libraries(my_lib eigenpy::eigenpy)
```

And `my lib` will get both the `-leigenpy` and the `-I/path/to/include/eigenpy`.

File: Config.cmake.in [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @PACKAGE_INIT@
 
+set("@PROJECT_NAME@_INCLUDE_DIRS" "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
+set("@PROJECT_NAME_UPPER@_INCLUDE_DIRS" "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
```
Comment (jmirabel) [inline Config.cmake.in]: What is the reason to include both ?

Comment (jcarpent) [reply Config.cmake.in]: To be robust and projects like Eigen for instance only define the UPPER version

**Comment by jcarpent** [comment]
@nim65s This is already done in EigenPy.

**Comment by jcarpent** [comment]
So @nim65s @jmirabel Do you think this is valuable to add it? I can handle it projectwise, but this would be painful to do for all projects that are related to ROS for instance ;)

**Comment by nim65s** [comment]
If this is required somewhere where we can't fix this, yes, sure, we can merge this PR, it won't harm us :)
But this is adding technical debt, so if it's not required, we should not do this.

**Comment by jcarpent** [comment]
Ok. Thanks @nim65s @jmirabel for your feedback. I will merge it now, as it seems useful for external projects.

**Comment by wxmerkt** [comment]
Thank you for the quick fix. What this does is add backwards compatibility - the changes in packaging recently broke existing workflows and downstream packages. This restores them. :-)

**Pull request merged**